docs/data/joy/getting-started/usage/usage.md

```diff
@@ -1,42 +1,42 @@
 # Usage
 
-<p class="description">Learn the basics of using Joy UI components in two quick steps.</p>
+<p class="description">Learn the basics of using Joy UI components.</p>
 
-## Getting started
+## Quickstart
 
-### Set up the `CssVarsProvider` component
-
-Go to your `App.js` file and replace it with the code snippet below.
-
-You should see the text `Hello from Joy` being rendered on your browser.
+The following code snippet demonstrates a simple app that uses the Joy UI [Button](/joy-ui/react-button/) component:
 
 ```jsx
+import * as React from 'react';
 import { CssVarsProvider } from '@mui/joy/styles';
+import Button from '@mui/joy/Button';
 
-function App() {
-  return <CssVarsProvider>Hello from Joy</CssVarsProvider>;
+export default function App() {
+  return (
+    <CssVarsProvider>
+      <Button variant="solid">Hello World</Button>
+    </CssVarsProvider>
+  );
 }
-
-export default App;
 ```
 
-### Render your first component
+You can play around with this code in the interactive Code Sandbox demo below.
+Try changing the `variant` on the Button to `soft` to see how the style changes:
 
-To render any Joy UI component, make sure you place them inside the `CssVarProvider` element.
+{{"demo": "Usage.js", "hideToolbar": true, "bg": true}}
 
-```diff
-import { CssVarsProvider } from '@mui/joy/styles';
-+ import Button from '@mui/joy/Button';
+### CssVarsProvider
 
-function App() {
-  return (
-    <CssVarsProvider>
-+     <Button>Joy UI</Button>
-    </CssVarsProvider>
-  );
-}
+In the example above, you can see that the Button component is nested within `<CssVarsProvider />`.
+This provider unlocks a whole host of customization options powered by CSS variables.
+See [Using CSS variables](/joy-ui/customization/using-css-variables/) for more details.
 
```

Comment:
I would have expected a `## Globals` on this page. From what I understand, we need to document the meta, the font and the CssBaseline. Actually, I see no CssBaseline in Joy UI: https://codesandbox.io/s/joy-ui-forked-j8qp8i?file=/index.tsx. How do we solve the problems that CssBaseline solve?

Reply:
Yeah, we should have CssBaseline in Joy UI as well. Added to the list.

Reply:
I skipped the "Globals" header here since it only contained one item. Would `<CssVarsProvider>` also be considered a "global"?